File: packages/svg-icons/src/icons/handle-resize-alt.ts
import { SVGIcon } from '../svg-icon.interface';

export const handleResizeAltIcon: SVGIcon = {
    name: 'handle-resize-alt',
    content: '<path stroke-linecap="round" stroke-linejoin="round" d="M19.5 12v7.5H12" fill="none"/>',
    viewBox: '0 0 24 24',
    variants: {
        'solid': '<path d="M19.5 12v7.5H12"/><path stroke-linecap="round" stroke-linejoin="round" d="M19.5 12v7.5H12z" fill="none"/>',
        'outline': '<path stroke-linecap="round" stroke-linejoin="round" d="M19.5 12v7.5H12" fill="none"/>',
        'duotone': '<path fill-opacity="0.2" d="M19.5 12v7.5H12"/><path stroke-linecap="round" stroke-linejoin="round" d="M19.5 12v7.5H12" fill="none"/>'
    },
    tags: ['handle', 'resize', 'alt', 'css', 'editing', 'scale', 'expand', 'shrink']
}

File: packages/svg-icons/src-cs/Telerik.SvgIcons/Icons/HandleResizeAlt.cs
namespace Telerik.SvgIcons
{
    public class HandleResizeAlt : SvgIconBase
    {
        public HandleResizeAlt()
        {
            Name = "handle-resize-alt";
            Content = "<path stroke-linecap=\"round\" stroke-linejoin=\"round\" d=\"M19.5 12v7.5H12\" fill=\"none\"/>";
            ViewBox = "0 0 24 24";
            Variants = new System.Collections.Generic.Dictionary<string, string>
            {
                { "solid", "<path d=\"M19.5 12v7.5H12\"/><path stroke-linecap=\"round\" stroke-linejoin=\"round\" d=\"M19.5 12v7.5H12z\" fill=\"none\"/>" },
                { "outline", "<path stroke-linecap=\"round\" stroke-linejoin=\"round\" d=\"M19.5 12v7.5H12\" fill=\"none\"/>" },
                { "duotone", "<path fill-opacity=\"0.2\" d=\"M19.5 12v7.5H12\"/><path stroke-linecap=\"round\" stroke-linejoin=\"round\" d=\"M19.5 12v7.5H12\" fill=\"none\"/>" }
            };
        }
    }
}
